runtime/js/40_http.js

```diff
@@ -81,8 +81,7 @@
       return {
         async next() {
           const reqEvt = await httpConn.nextRequest();
-          if (reqEvt === null) return { value: undefined, done: true };
-          return { value: reqEvt, done: false };
+          return { value: reqEvt, done: reqEvt === null };
```

Comment:
Can you add a comment here about that deopt so the next person doesn't rewrite it by accident?

Reply:
Added comment